src/llmq/quorums_chainlocks.cpp

```diff
@@ -149,41 +151,36 @@ void CChainLocksHandler::ProcessNewChainLock(NodeId from, const llmq::CChainLock
         const CBlockIndex* pindex = blockIt->second;
         bestChainLockWithKnownBlock = bestChainLock;
         bestChainLockBlockIndex = pindex;
+        if (lastNotifyChainLockBlockIndex != bestChainLockBlockIndex) {
+            lastNotifyChainLockBlockIndex = bestChainLockBlockIndex;
+            pindexNotify = pindex;
+        }
     }
 
-    EnforceBestChainLock();
+    if (pindexNotify) {
```

Comment:
I know we do this everywhere in the codebase, but I hate how we do this automatic integer/pointer to boolean conversions. 
Would prefer `if (pindexNotify != 0)`

Reply:
fixed :)